tests/_bootstrap.php

```diff
@@ -40,3 +40,23 @@
 // By default, do not enable the Custom Tables v1 implementation in tests.
 putenv( 'TEC_CUSTOM_TABLES_V1_DISABLED=1' );
 $_ENV['TEC_CUSTOM_TABLES_V1_DISABLED'] = 1;
+
+/**
+ * Ensure coupons are loaded in the order modifiers.
+ *
+ * Modify the global variable directly, because WordPress hasn't loaded at this point.
+ *
+ * @see WP_Hook::build_preinitialized_hooks()
+ */
+if ( ! array_key_exists( 'wp_filter', $GLOBALS ) ) {
+	$GLOBALS['wp_filter'] = [];
+}
+
+$GLOBALS['wp_filter']['tec_tickets_order_modifiers_coupons_enabled'] = [
+	10 => [
+		[
+			'function'      => '__return_true',
+			'accepted_args' => 1,
+		],
+	],
+];
```

Comment:
Why not use a constant or ENV variable like the other feature flags?

Reply:
Because it was the easiest way to keep all of the coupon functionality turned off until it is ready. This filter will be removed once the coupons functionality is ready.